### ISSUE TITLE
Track EX Eligibility Of Gym

### ISSUE THREAD

**Issue by dandesousa** [PokeNavBot/PokeNav#106]
```
moriakaice08/07/2018
Feature request: ability for the bot to track gyms that triggered EX raids, later on provide stats on those (and a list of ex-eligible raids by the principle of hosting EX raid in the past).

theENDtype08/07/2018
@moriakaice  I would like to see something like this with maybe the $gym-info cmd. Have an option for EX Gym: flag that is defaulted to No. Admins can just update their DBs for each gym to Yes to mark it an EX gym on their server.

moriakaice08/07/2018
@theENDtype I don't even need to manual marking/marking of "potential" gyms - it'd be enough to just track the already existing EX gyms, especially with communities that have multiple of them (like ours in London)
```

**Comment by fortepc** [comment]
So, like an "Reported EX Raids" counter on a gym that increments every time an EX raid pass is reported at that gym. To avoid mistakes, it should only actually increment if that EX channel actually gets used, so if the raid channel is deleted before the raid time it doesn't increment. Then that number of EX Raids at that location will show in the `#active-raids` entry that just highlights "Reported EX Raids at this Gym: 3"

**Comment by emman31** [comment]
With the new feature that tag exraid eligible gym, if the tag can be seen in the screenshot, maybe that can be used for recognizing eligibility?

**Comment by fortepc** [comment]
Yes! So instead of tracking it based on ex raids that get reported, just have it be a true/false flag and the OCR / Computer Vision just checks for if that visible EX Raid flag is present and marks that gym's metadata as EX Eligible, and shows that specific raid as EX eligible in the raid metadata

**Comment by fortepc** [comment]
Must be a bug.

**Comment by dandesousa** [comment]
Its possible this will be fixed -- I've also noticed that ex eligible gyms are sometimes not marked ex eligible even though they should be (maybe its based on eligibility that week?) We should probably let things shake out a bit more and then decide how best to do this. 

I think depending on the implementation I would rely entirely on the presence of the label or mark it in the metadata for the gym and show it every time.

**Comment by fortepc** [comment]
Now that you're working with pokemongomap.info, maybe they would be interested in adding that ex eligibility to their metadata.

**Comment by dandesousa** [comment]
I am not working with pokemongomap.info, they are not interested in an integration at this time.

**Comment by fortepc** [comment]
oh :( sad. 

Well if it makes any difference, they will be adding EX Elegibility
https://twitter.com/David28227/status/1047689631905603584 
![image](https://user-images.githubusercontent.com/26149601/46571864-abaffb00-c94a-11e8-8f39-c016a5dfe528.png)
